### PR TITLE
♻️ Use Storybook `args` (first round)

### PR DESCRIPTION
**Pull request by alanorozco**
This is the first step to replace Storybook [Knobs (deprecated)](https://www.npmjs.com/package/@storybook/addon-knobs) with [Controls.](https://storybook.js.org/docs/react/essentials/controls)

We have over 60 Storybook files. I'm submitting the PRs in multiple rounds to prevent overwhelming the reviewers.

In summary:

1. Installs `@storybook/addon-controls`
2. Updates template for generated Stories.
3. Updates 15 Stories across extensions.

### CODE REVIEW

File: extensions/amp-iframe/1.0/storybook/Basic.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 import * as Preact from '#preact';
 import {Iframe} from '../component';
-import {text, withKnobs} from '@storybook/addon-knobs';
```
Comment (caroqliu) [inline extensions/amp-iframe/1.0/storybook/Basic.js:3]: The knobs tab will go away once we remove the addon line in the main.js files right?
Screen Shot 2021-09-02 at 10 04 26 AM

File: extensions/amp-instagram/1.0/storybook/Basic.js [hunk 2]
```diff
@@ -13,29 +11,25 @@ import {Instagram} from '../component';
 export default {
   title: 'Instagram',
   component: Instagram,
-  decorators: [withKnobs],
+  args: {
+    width: 500,
+    height: 600,
+    shortcode: 'B8QaZW4AQY_',
+    captioned: false,
```
Comment (caroqliu) [inline extensions/amp-instagram/1.0/storybook/Basic.js:18]: This new boolean ui is way nicer than the checkbox :) 
Screen Shot 2021-09-02 at 10 05 15 AM

**Review by caroqliu — APPROVED** [review]
Thanks for doing this Alan!

File: extensions/amp-iframe/1.0/storybook/Basic.js (hunk 1 above)
Comment (alanorozco) [reply extensions/amp-iframe/1.0/storybook/Basic.js:3]: That's correct.

File: build-system/tasks/make-extension/template/shared/extensions/amp-__component_name_hyphenated__/__component_version__/storybook/Basic.amp.js [hunk 3]
```diff
@@ -1,32 +1,29 @@
...
-      example-property="example string property value"
+      {...args}
     >
       This text is inside.
     </amp-__component_name_hyphenated__>
   );
 };
-
-ExampleUseCase.story = {
-  name: 'Example use case story'
```
Comment (samouri) [inline build-system/tasks/make-extension/template/shared/extensions/amp-__component_name_hyphenated__/__component_version__/storybook/Basic.amp.js:31]: FMI: do we lose out on the name?

File: extensions/amp-social-share/1.0/storybook/Basic.amp.js [hunk 4]
```diff
@@ -1,87 +1,77 @@
...
 
 import * as Preact from '#preact';
 
+/*
+ * amp-social-share allows the user to set various parameters to configure
+ * its behavior.  These parameters are controlled by storybook controls and
+ * are summarized below:
+ *
+ * Key Configuration Parameters -
+ * type - This is a required attribute.  It configures a pre-configured set
```
Comment (samouri) [inline extensions/amp-social-share/1.0/storybook/Basic.amp.js:11]:
```suggestion
 * type - This is a required attribute.  It configures a set
```

File: extensions/amp-social-share/1.0/storybook/Basic.amp.js [hunk 5]
```diff
@@ -1,87 +1,77 @@
...
+ *   of parameters needed to share with a particular social-media.  Most
```
Comment (samouri) [inline extensions/amp-social-share/1.0/storybook/Basic.amp.js:12]:
```suggestion
 *   of parameters to share with a particular social-media.  Most
```

File: extensions/amp-social-share/1.0/storybook/Basic.amp.js [hunk 6]
```diff
@@ -1,87 +1,77 @@
...
+ *   notably, the share-endpoint is set by the type.  Configuration details
+ *   can be found in amp-social-share-config.js.  The user may also select
+ *   'custom' with this knob to manually specify the required parameters.
```
Comment (samouri) [inline extensions/amp-social-share/1.0/storybook/Basic.amp.js:15]:
```suggestion
 *   'custom' with this control to manually specify the required parameters.
```

**Review by samouri — APPROVED** [review]

File: build-system/tasks/make-extension/template/shared/extensions/amp-__component_name_hyphenated__/__component_version__/storybook/Basic.amp.js (hunk 3 above)
Comment (alanorozco) [reply build-system/tasks/make-extension/template/shared/extensions/amp-__component_name_hyphenated__/__component_version__/storybook/Basic.amp.js:31]: Renders as `Example Use Case`. It's nicer to not specify it if it's they're equal, or nearly so.

**Pull request merged**